### DISCUSSION TITLE
Meross clarification

### DISCUSSION THREAD

**Discussion by Cinett87** [theyosh/TerrariumPI#747] [Q&A] [answered]
I have added in my credentials into the settings, but I am not understanding how to add a meross power switch to the system. scan for relay returns nothing and I have attempted every form of address and nothing is registering with it.

**Comment by theyosh** [comment, marked as answer]
Hmm, if you have added your switches to you account with the phone app, then after a restart you should have the new relays in TP4.

But, it could be that you have a switch type which I did not implement. So what kind of relay do you use?

**Reply by Cinett87** [reply]
its the 4 outlet me ross surge protector like you have as an example picture

**Reply by Cinett87** [reply]
it finally worked I had it added to home app but not meross app that was user error
